guides/common/modules/ref_customizing-the-registration-templates.adoc

```diff
@@ -21,10 +21,13 @@ You can configure the following global parameters by navigating to *Configure* >
 
 ifndef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `false`.
+Set the parameter type to `boolean` to override the parameter value.
 endif::[]
 ifdef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `true`.
+Set the parameter type to `boolean` to override the parameter value.
 endif::[]
+If the parameter is set to `false`, it prevents hosts from appearing in the Red{nbsp}Hat Inventory reports and also the Insights client from uploading reports to the Inventory of your {RHCloud}.
```

Comment:
Should this say "Red Hat Insights" instead of "Red Hat Inventory reports"? @mjivraja

Reply:
Because when I look into the WebUI, there are the following items under *Configure* > *RH Cloud*: *Inventory Upload* and *Insights*.

Reply:
```suggestion
If the parameter is set to `false`, it prevents hosts from appearing in the Red{nbsp}Hat Insights and also the Insights client from uploading Inventory reports to your {RHCloud}.
```
Maybe like this? Pls, check this with SMEs.

Reply:
@ShimShtein perhaps you can enlighten us on this.

Reply:
I guess, hosts are still part of RH Insights despite the parameter being false. But definitely, let's wait for @ShimShtein

Reply:
The inventory on the cloud side accepts many types of reports. As long as at least one reporter continues to report about a host, it will appear in the cloud inventory.
Here we are talking about two reporters that are controlled by Satellite: the rh_cloud (Satellite) generated reports and `insights-client` reports. While those are the main contributors to the inventory, they are not the only ones that exist, and that's why I am a bit reluctant about documenting this parameter as something that will prevent the entry in the inventory for sure. We want to indicate that this parameter changes the behavior of those two reports.

Reply:
Maybe something along the lines of
```
It prevents Satellite and the Insights client from uploading Inventory reports to your {RHCloud}.
```
Does this make sense?